src/content/get-started/ch2-add-commands.md

```diff
@@ -0,0 +1,514 @@
+---
+title: Making Your CLI Program Interactive
+description: Add simple commands to your cli program. Learn the fundamentals of Dart syntax including control flow, collections, variables, functions, and more.
+---
+
+In this chapter, you'll get hands-on with Dart syntax. You'll cover control
+flow, collections, variables, functions, and more. You'll learn how to read user
+input, print usage information, and create a basic command-line interaction. By
+the end, you'll have a foundational understanding of Dart syntax.
+
+## Prerequisites
+
+* Completed Chapter 1 and have a working Dart development environment.
+* Familiarity with basic programming concepts (variables, data types, control
+  low).
```

Comment:
```suggestion

Before working through this section, we suggest that you first:

* Complete Chapter 1 and have a working Dart development environment.
* Have a familiarity with basic programming concepts 
  (such as variables, data types, control flow).
```

Reply:
I think we should be stronger here, because the code from previous chapters is required to continue in the next chapters.

How about "Before you begin this chapter, ensure you have:"